mail.py

```diff
@@ -57,9 +56,8 @@
 students = csv.reader(file)
 
 def setup_server():
-    server = smtplib.SMTP("smtp.gmail.com", 587)
+    server = smtplib.SMTP("smtp-relay.gmail.com", 587)
     server.starttls()
-    server.login(args.from_addr, os.environ["LIONMAIL_DEVICE_PASS"])
```

Comment:
This is the one thing I'm a little concerned about, since there may be people that use this off of CLAC (since this is a public repo), and as such would need this line. I'm cool with getting rid of this since this script doesn't really get used that much anymore, but we should open an issue just to make a note of this.